docs/algorithms/mapper.rst

```diff
@@ -14,12 +14,13 @@ The mapper uses a library (hash table) to facilitate Boolean matching.
 For technology mapping, it needs `tech_library` while for graph mapping
```

Comment:
(GitHub doesn't let me comment on unchanged lines so I put it here)
Could we change the title of this page to "Technology mapping and network conversion" so that it's easier for totally-new people to find what they want from the titles?

Reply:
Sure, good idea!